eng/scripts/Get-TypeSpec-Folders.ps1

```diff
@@ -42,7 +43,12 @@ else {
 $typespecFolders = @()
 foreach ($file in $changedFiles) {
   if ($file -match 'specification\/[^\/]*\/') {
-    $typespecFolder = (Get-ChildItem -path $matches[0] tspconfig.yaml -Recurse).Directory.FullName | ForEach-Object {if ($_) { [IO.Path]::GetRelativePath($($pwd.path), $_) }}
+    $typespecFolder = (Get-ChildItem -path $matches[0] tspconfig.yaml -Recurse).Directory.FullName
+    | ForEach-Object {if ($_) { [IO.Path]::GetRelativePath($($pwd.path), $_) }}
+
+    if ($ci) {
+      $typespecFolder = $typespecFolder | Where-Object {!(Get-ChildITem -force -path $_ .skip-typespec-validation)}
```

Comment:
Is the expectation that people put this empty marker file next to their tspconfig?

Reply:
yes that's the design so far. We thought about putting it in the tspconfig but it doesn't quite fit into the definition of "configuring tsp projects"